src/components/habits/habits.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import {Button, TextField } from '@material-ui/core';
+
+import './habits.css'
+import AddHabit from '../addHabit';
+
+
+class Item extends React.Component {
+    constructor (props){
+        super ();
+
+        this.state = {
+            checked: false
+        };
+
+        this.handleClick = this.handleClick.bind(this);
+    }
+    handleClick (e){
+        this.setState({
+            checked: !this.state.checked
+        });
+
+    }
+
+    render (){
+
+        let text = this.state.checked ? <strike>{this.props.message}</strike> : this.props.message;
+        return (
+            <div className="row">
+                <div className="col-md-12">
+                    <input type="checkbox" onClick={this.handleClick} />&nbsp;{text}
```

Comment:
put text into label, and add `htmlFor` there so you can click on label too. It improves ux.